src/Behat/Behat/Definition/Printer/ConsoleDefinitionInformationPrinter.php

```diff
@@ -39,12 +39,37 @@ public function setSearchCriterion($criterion)
      * {@inheritdoc}
      */
     public function printDefinitions(Suite $suite, $definitions)
+    {
+        $this->printDefinitionsWithOptionalSuite($definitions, $suite);
```

Comment:
This printer assumed that a Suite was available. We modify it so that a suite becomes optional